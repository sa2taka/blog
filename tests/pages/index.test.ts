import Vuetify from 'vuetify';
import { createLocalVue, shallowMount } from '@vue/test-utils';
import Index from '@/pages/index.vue';

const localVue = createLocalVue();

jest.mock('contentful', () => ({
  createClient: (_: any) => {},
}));

describe('pages/index', () => {
  let vuetify: any;
  beforeEach(() => {
    vuetify = new Vuetify();
  });
  const mountWithVuetify = () => {
    return shallowMount(Index, {
      localVue,
      vuetify,
    });
  };

  test('should render', () => {
    const wrapper = mountWithVuetify();
    expect(wrapper.isVueInstance()).toBeTruthy();
    expect(wrapper).toMatchSnapshot();
  });
});
