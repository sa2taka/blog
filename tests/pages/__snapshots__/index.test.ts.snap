// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/index should render 1`] = `
<v-layout column="" justify-center="" align-center="">
  <posts-stub></posts-stub>
</v-layout>
`;
